client/app/bundles/course/group/pages/GroupShow/GroupManager/GroupUserManagerList.jsx

```diff
@@ -81,12 +102,47 @@ const styles = {
   },
 };
 
+const GroupUserManagerListItemChoice = ({ user, onChangeDropdown }) =>
+  user.role !== 'student' ? (
```

Comment:
As much as possible, we try to avoid negative conditionals. So u can swap it to `user.role == 'student'`

Reply:
okay